<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstrucProv</class>
 <widget class="QDialog" name="ConstrucProv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Monto Bruto</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_cerca">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerca y puertas</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_caseta">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Caseta y veladores</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_oficina">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Oficina</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bodega">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bodega cubierta</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_dormit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dormitorios</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_sanitario">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sanitarios</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_comedores">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Comedores y cocinas</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_hidra">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Inst. hidráulicas</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_elect">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Inst. eléctricas</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_camino">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Camino acceso</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_agregar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>+ Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_regresar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- Regresar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cerca">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_caseta">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oficina">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bodega">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dormit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sanitarios">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_comedores">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_hidra">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_elect">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_camino">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
